README.md

```diff
@@ -60,3 +61,11 @@ check` before proceeding to debug using `bin/webpack`.
 **If assets work in dev but not in tests**, first confirm that you can compile
 by invoking `bin/webpack`. If all is well, there is a chance that
 `public/packs-test` contains stale output. Delete it and re-run the suite.
+
+## Documentation
```

Comment:
`Documentation` now has equal status with `Webpacker` and `Docker workflow`. Should we demote those to be under a `Development process` heading or sth?